Makefile

```diff
@@ -0,0 +1,89 @@
+###
+# This Makefile provides portable, stack-agnostic commands (unlike npm)
+#
+# These commands are meant to be run from your local machine, not from
+# within the docker container. Each command handles exec'ing into the
+# docker container to run the command
+
+ifneq (,)
+	This makefile requires GNU Make.
+endif
+# #######
+# Basics
+platform-onboarded:
+	make migrate-docker
+	docker exec -it platform_platform_1 /bin/bash
+	docker exec -it platform_platform_1 npm run dev
+
+platform-exec:
+	@docker exec -it platform_platform_1 bash
+exec:platform-exec
+
+# ######
+# DB
+
+
+seed-docker:
+	docker exec -it platform_platform_1 /bin/bash
+	composer migrate
+
+seed:
+	make seed-docker
+
+
+migrate:
+	make migrate-docker
+
+
+migrate-docker:
+	docker exec -it platform_platform_1 php artisan migrate:refresh -vvv
+
+db-total-reset:
+	@echo "🔥 Resetting DB"
+	docker exec -it platform_platform_1 composer dump-autoload
+	make migrate-docker
+	make seed-docker
```

Comment:
doesn't migrate-docker already run the migrations?  seed-docker is just running `composer migrate` as well